doc/examples/cloud-config-ansible-controller.txt

```diff
@@ -0,0 +1,120 @@
+#cloud-config
+#
+# Uses deploy keys to pull playbooks from a remote repo and run
+# those plays.
+#
+
+packages_update: true
+packages_upgrade: true
+packages:
+  - git
+  - python3-pip
+
+users:
+- name: ansible
+  gecos: Ansible User
+  shell: /bin/bash
+  groups: users,admin,wheel,lxd
+  sudo: ALL=(ALL) NOPASSWD:ALL
+
+lxd:
```

Comment:
I think this example could use more comments as to what exactly is happening and why LXD is being used here. I couldn't understand just from looking at the example. I had to figure it out with the integration test.

Reply:
+1 Updated